### PR TITLE
Fix grammar in sentence regarding return values of a View function

### PR DESCRIPTION
**Pull request by clay-aptos**

### CODE REVIEW

File: developer-docs-site/docs/guides/aptos-apis.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ A view function request may look like this in the Typescript SDK:
     expect(balance[0]).toBe("100000000");
 ```
 
-The view function is instead a list of return values as a vector. The computation results are returned in a JSON file, as with all other Move value queries. Or you can receive the output as in a Binary Canonical Serialization (BCS) encoded format.
+The view function instead returns a list of values as a vector. The computation results are returned in a JSON file, as with all other Move value queries. Or you can receive the output as in a Binary Canonical Serialization (BCS) encoded format.
 
 The user sends a payload containing the address, module, and function name of the Move view function on chain along with serialized arguments.  We execute it and get the execution result back to the user. For example, a function may reveal the balance of an account with a return value in the form of an integer. The View function requires being able to query the on-chain state in Aptos.
```
Comment (movekevin) [inline developer-docs-site/docs/guides/aptos-apis.md:61]: More nits:
1. There's double spacing before "We execute it".
2. Does "we" make sense here? Who's we?

**Review by movekevin — COMMENTED** [review]
A few more sentences that don't quite make sense
1. "the results are read values on the blockchain" on line 44
2. "and the Aptos Simulation API is expected write output on the blockchain." also on line 44

File: developer-docs-site/docs/guides/aptos-apis.md (hunk 1 above)
Comment (clay-aptos) [reply developer-docs-site/docs/guides/aptos-apis.md]: Fixed by removing space and replacing "We" with "Aptos"

**Comment by clay-aptos** [comment]
> A few more sentences that don't quite make sense
> 
> 1. "the results are read values on the blockchain" on line 44
> 2. "and the Aptos Simulation API is expected write output on the blockchain." also on line 44

Ah, this text was introduced by @gregnazario in:
https://github.com/aptos-labs/aptos-core/pull/6315/commits/7ac2201d04cdf4c2d738436b927ae2fa5a908ee3

So I will let him explain what he meant for you here or in another commit. I originally wrote that sentence to read, "Using the Aptos Simulation API, the results are the entire side effects of the transaction."

**Review by gregnazario — APPROVED** [review]
Let's get what you have in and please I encourage @movekevin to make these updates directly, it takes a couple minutes and can be done entirely from the browser

**Pull request merged**